atom/browser/api/atom_api_content_tracing.cc

```diff
@@ -116,7 +116,7 @@ void OnTraceBufferUsageAvailable(atom::util::Promise promise,
                                  size_t approximate_count) {
   mate::Dictionary dict = mate::Dictionary::CreateEmpty(promise.isolate());
   dict.Set("percentage", percent_full);
-  dict.Set("value", approximate_count);
+  dict.Set("value", static_cast<uint64_t>(approximate_count));
```

Comment:
I'm curious what lint rule triggered this, because the old version seems fine to me here.

Reply:
there was a converter from `unsigned long`, which was used for `size_t`

Reply:
I wasn't sure what to do here. I can keep the converter and just silence the warnings about `unsigned long`

Reply:
I've reverted this change and kept the `unsigned long` converter as there were some build errors on Windows. Added `// NOLINT` there for now